lib/github.js

```diff
@@ -143,6 +143,32 @@ class GitHub {
     })
   }
 
+  getPr (pr) {
+    const owner = this.config.owner
+    const repo = this.config.repo
+
+    return new Promise((resolve, reject) => {
+      const url = `https://api.github.com/repos/${owner}/${repo}/pulls/${pr}`
+      if (process.env.VERBOSE) {
+        console.log(`About to send GET to ${url}`)
+      }
+      fetch(url, getFetchOpts())
+        .then((resp) => {
+          if (!resp.ok) {
+            reject(`Error: ${resp.status}: ${JSON.stringify(resp.json, null, 2)}`)
+            return
+          }
+
+          resp.json().then((json) => {
+            resolve(json)
+          })
+        })
+        .catch((err) => {
+          reject(err)
+        })
+    })
+  }
```

Comment:
Whenever I see `reject` and `resolve` in a function, it usually means that the code is not doing optimum promise-composition. 

And, if fact, I think that's true with this code. How about this approach instead:
```javascript
const url = `https://api.github.com/repos/${owner}/${repo}/pulls/${pr}`
if (process.env.VERBOSE) {
  console.log(`About to send GET to ${url}`)
}
return fetch(url, getFetchOpts())
  .then((resp) => {
    if (!resp.ok) {
      throw new Error(`${resp.status}: ${JSON.stringify(resp.json, null, 2)}`)
    }
    return resp.json().then((json) => json)
  })
```